### DISCUSSION TITLE
Great initiative!

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/jats-from-markdown#2] [General]
Hey,

great to see this happen here! I've been using quarto for two weeks now building everything that can be built with it and finally thought: If it could also generate JATS it would be nearly perfect. @tarleb @arfon Have you heard anything about this at JATS con?

**Comment by tarleb** [comment]
The only info is basically me saying "I really want to get it working with Quarto", but nothing else yet. It's on my todo list.
